core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -3,18 +3,18 @@
      ng-class="{'faded-topic-card': !$ctrl.isPublished()}">
   <a ng-href="<[$ctrl.getTopicPageUrl()]>">
     <div class="topic-thumbnail">
-      <picture>
+      <span class="topic-coming-soon" ng-if="!$ctrl.isPublished()" translate="I18N_CLASSROOM_PAGE_COMING_SOON"></span>
+      <picture class="d-flex" ng-class="{'grayscale-effect': !$ctrl.isPublished(), 'margin-top-offset': !$ctrl.isPublished()}">
```

Comment:
We have a bunch of flexbox classes in oppia.css (all the way at the bottom of the file). Can you check once if those can be used here or not?

Reply:
Ah ok. I have used `fx-row` instead. Done.